.env.example

```
@@ -28,3 +28,9 @@ APPLE_TEAM_ID=""
 APPLE_CLIENT_ID=""
 APPLE_KEY_ID=""
 APPLE_PRIVATE_KEY=""
+
+# Vercel AI Gateway
+AI_GATEWAY_API_KEY=""
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)